runtime/src/bank.rs

```diff
@@ -2054,6 +2088,215 @@ mod tests {
         bank
     }
 
+    #[test]
+    fn test_rent_distribution() {
```

Comment:
The length of this unit test is mildly terrifying to me (although I appreciate the zeal!).  All the validator_N_ variables look rip for consolidating into an array of structs for example.

Reply:
@mvines yeah, let me try to restructure it.

Reply:
@mvines Tried, too much hassle, as fields are interdependent.